### PR TITLE
PLNSRVCE-813: Expand pipelineRun in the PR

### PR DESCRIPTION
**Pull request by Michkov**
For HACBS extract Pipeline from bundle and pass it to pipelineRun as pipelineRef

### CODE REVIEW

**Comment by Michkov** [comment]
Created PR -> https://github.com/Michkov/devfile-sample-python-basic/pull/8/files

File: go.mod [hunk 1]
```diff
@@ -18,6 +18,12 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/docker/cli v20.10.12+incompatible // indirect
```
Comment (mmorhun) [inline go.mod:22]: Please keep all indirect dependencies under the same require

File: controllers/component_build_controller.go [hunk 2]
```diff
@@ -237,9 +240,20 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
...
 		}
 
-		// Manage merge request for Pipelines as Code configuration
 		bundle := gitopsConfig.BuildBundle
-		mrUrl, err := ConfigureRepositoryForPaC(component, pacSecret.Data, webhookTargetUrl, webhookSecretString, bundle)
+
+		// Get PipelineSpec from bundle for HACBS workflow
+		var pipelineSpec tektonapi.PipelineSpec
+		if gitopsConfig.IsHACBS {
+			if pipelineSpec, err = resolvePipelineSpec(bundle, defaultPipelineName); err != nil {
```
Comment (mmorhun) [inline controllers/component_build_controller.go:248]: Please add a comment here

**Review by mmorhun — APPROVED** [review]

File: controllers/component_build_controller.go (hunk 2 above)
Comment (Michkov) [reply controllers/component_build_controller.go:250]: added

**Pull request merged**